tests/virt/node/general/test_machinetype.py

```
@@ -254,5 +278,9 @@ def test_machine_type_as_rhel_9_6(machine_type_from_kubevirt_config):
     ],
     indirect=True,
 )
-def test_legacy_machine_type(updated_kubevirt_config_machine_type, vm_for_legacy_machine_type_test):
-    validate_machine_type(vm=vm_for_legacy_machine_type_test, expected_machine_type=MachineTypesNames.pc_i440fx_rhel7_6)
+def test_legacy_machine_type(updated_kubevirt_config_machine_type, vm_for_legacy_machine_type_test, admin_client):
```

Ruff (0.15.0)

[warning] Unused function argument: `updated_kubevirt_config_machine_type`

(ARG001)